lib/cucumber/html_builder.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+require 'builder'
+
+module Cucumber
+  class HtmlBuilder < Builder::XmlMarkup
```

Comment:
I think it would be best if this lives in the `Cucumber::Formatter` namespace so it sits alongside `formatter/html.rb`